src/test/java/teammates/test/cases/webapi/UpdateInstructorActionTest.java

```diff
@@ -83,7 +82,9 @@ protected void testExecute() {
         assertEquals(HttpStatus.SC_BAD_REQUEST, actionOutput.getStatusCode());
 
         msg = (MessageOutput) actionOutput.getOutput();
-        String expectedErrorMessage = FIELD_VALIDATOR.getInvalidityInfoForEmail(invalidEmail);
+
```

Comment:
Remove these 2 unnecessary newlines

Reply:
@amrut-prabhu Thanks for the review. Apologies of the mistakes, I have corrected all the reviewed corrections.